sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -4,10 +4,13 @@
 
 #### Features Added
 * Added support for Move operation - See [PR 31078](https://github.com/Azure/azure-sdk-for-java/pull/31078)
+* GA of subpartitioning functionality in SDK - See [32501](https://github.com/Azure/azure-sdk-for-java/pull/32501)
+* Added ability for SDK to use partial partition keys for queries in subpartitioned containers - See [32501](https://github.com/Azure/azure-sdk-for-java/pull/32501)
 
 #### Breaking Changes
 
 #### Bugs Fixed
+* Fixed `readMany` API to take in hierarchical partition keys - See [32501](https://github.com/Azure/azure-sdk-for-java/pull/32501)
```

Comment:
i think with the current fix, we are going to throw an exception if any partial partition key is provided, so probably should remove this entry.